transformer_engine/pytorch/fp8.py

```diff
@@ -35,6 +37,10 @@
 _dp_amax_reduce_forward_idx = 0
 _dp_amax_reduce_backward_idx = 0
 
+jit_fuser = torch.jit.script
+if packaging.version.Version(version("torch")) >= packaging.version.Version("2"):
```

Comment:
Why not use something like `torch.__version__`?

Reply:
Because `torch.__version__` is of specific type `TorchVersion` which makes comparisons non-trivial to do.

Reply:
From what I can tell, `TorchVersion` has a pretty convenient interface:
```python
if torch.__version__ >= "2":
```
See: https://github.com/pytorch/pytorch/blob/32360b48e8d09984beb831ccb4a9d91b2de47189/torch/torch_version.py#L41